src/GuiBase/Viewer/Viewer.hpp

```diff
@@ -120,6 +121,12 @@ class RA_GUIBASE_API Viewer : public QWindow {
     /// Moves the camera so that the whole scene is visible.
     void fitCameraToScene( const Core::Aabb& sceneAabb );
 
+    /// Save the camera state to a file
+    void saveCamera( std::ostream& out ) const;
+
+    /// Load the camera state from a fil
+    void loadCamera( std::istream& in );
+
```

Comment:
See comment above ... put this on a plugin or jutify clearly why these methods are mandatory.